src/networks.js

```diff
@@ -14,6 +15,7 @@ module.exports = {
     dustThreshold: 546 // https://github.com/bitcoin/bitcoin/blob/v0.9.2/src/core.h#L151-L162
   },
   testnet: {
```

Comment:
Actually, I think for this change to make sense,  it should be named `testnet3`.
If `testnet` is ever to change,  then the `genesisBlockId` will also change.

I see nothing wrong with aliasing `testnet` to the "latest" network however.
But `testnet3` should be independent IMHO...